lib/service/Service.ts

```diff
@@ -1,13 +1,14 @@
 import Logger from '../Logger';
 import Pool from '../p2p/Pool';
 import OrderBook, { OrderArrays } from '../orderbook/OrderBook';
-import LndClient from '../lndclient/LndClient';
+import LndClient, { LndClientConfig } from '../lndclient/LndClient';
 import RaidenClient, { TokenSwapPayload } from '../raidenclient/RaidenClient';
 import { OwnOrder } from '../types/orders';
 import Config from '../Config';
 import { EventEmitter } from 'events';
 import SocketAddress from '../p2p/SocketAddress';
 import errors from './errors';
+import lnderrors from '../lndclient/errors';
```

Comment:
Rename this `lndErrors` to match naming used elsewhere like the errors imports in GrpcService.ts.

Reply:
done